4-forms-validations/form.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Html Form</title>
+
+    <style>
+      body {
+        display: flex;
+        flex-direction: column;j
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+      }
+      form {
+        display: flex;
+        flex-direction: column;
+      }
+      label {
+        margin: 7px;
+      }
+      .hobbies {
+        display: flex;
+        flex-direction: column;
+      }
+      fieldset {
+        margin-block: 5px;
+      }
+    </style>
+  </head>
+  <body>
+    <header><h1>Simple Form</h1></header>
+    <main>
+      <form>
+        <label for="name"
+          >First Name <br />
+          <input type="text" name="first-name" id="fn" required />
+        </label>
+        <label for="lastname">
+          Last Name <br />
```

Comment:
If you are using CSS styles I would avoid the `<br /`> and in this case, I would use CSS styles instead. 

_"In general, <br> tags are used where line breaks are syntactically relevant, such as poetry (where a line break determines the structure/flow of the actual content, vs the appearance)."_

Take a look to this StackOverflow discussion: https://stackoverflow.com/questions/3937515/when-to-use-br-line-breaks-vs-css-positioning